ads/_config.js

```diff
@@ -185,6 +185,16 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  appvador: {
+    prefetch: [
```

Comment:
We provide a `renderStart` and `noContent` API that help to provide better UX. We highly recommend implementing them. 
Please refer to https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis and let me know if there's any question. Thank you.

Reply:
@adg-dev did you get into this?

Reply:
@lannka add this on last commit. please check again.